etc/start_faux

```diff
@@ -40,6 +40,7 @@ else
 
     ip route add $IPADDR/16 via $GATEWAY dev $IFACE
     ip route add default via $GATEWAY
+    ping $GATEWAY &
```

Comment:
I'm not sure this is a good idea since it will horribly pollute the logs, no?

Reply:
Changed this so that static ip device only ping during fot tests.